From: Debian PHP Maintainers <pkg-php-maint@lists.alioth.debian.org>
Date: Sat, 2 May 2015 10:26:50 +0200
Subject: libtool_fixes

---
 TSRM/configure.in | 3 ---
 configure.in      | 2 --
 2 files changed, 5 deletions(-)

Index: dotdeb-php/TSRM/configure.in
===================================================================
--- dotdeb-php.orig/TSRM/configure.in
+++ dotdeb-php/TSRM/configure.in
@@ -17,9 +17,6 @@ TSRM_BASIC_CHECKS
 TSRM_THREADS_CHECKS
 	
 AM_PROG_LIBTOOL
-if test "$enable_debug" != "yes"; then
-  AM_SET_LIBTOOL_VARIABLE([--silent])
-fi
 
 dnl TSRM_PTHREAD
 
Index: dotdeb-php/configure.in
===================================================================
--- dotdeb-php.orig/configure.in
+++ dotdeb-php/configure.in
@@ -1420,8 +1420,6 @@ AC_PROVIDE_IFELSE([PHP_REQUIRE_CXX], [],
 ])
 AC_PROG_LIBTOOL
 
-PHP_SET_LIBTOOL_VARIABLE([--silent])
-
 dnl libtool 1.4.3 needs this.
 PHP_SET_LIBTOOL_VARIABLE([--preserve-dup-deps])
 
