From: Debian PHP Maintainers <pkg-php-maint@lists.alioth.debian.org>
Date: Sat, 2 May 2015 10:26:57 +0200
Subject: hack-phpdbg-to-explicitly-link-with-libedit

---
 sapi/phpdbg/config.m4 | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

Index: dotdeb-php/sapi/phpdbg/config.m4
===================================================================
--- dotdeb-php.orig/sapi/phpdbg/config.m4
+++ dotdeb-php/sapi/phpdbg/config.m4
@@ -25,7 +25,9 @@ if test "$BUILD_PHPDBG" == "" && test "$
   PHP_PHPDBG_FILES="phpdbg.c phpdbg_parser.c phpdbg_lexer.c phpdbg_prompt.c phpdbg_help.c phpdbg_break.c phpdbg_print.c phpdbg_bp.c phpdbg_opcode.c phpdbg_list.c phpdbg_utils.c phpdbg_info.c phpdbg_cmd.c phpdbg_set.c phpdbg_frame.c phpdbg_watch.c phpdbg_btree.c phpdbg_sigsafe.c phpdbg_wait.c phpdbg_io.c phpdbg_eol.c phpdbg_out.c"
 
   if test "$PHP_READLINE" != "no" -o  "$PHP_LIBEDIT" != "no"; then
-  	PHPDBG_EXTRA_LIBS="$PHP_READLINE_LIBS"
+#  	PHPDBG_EXTRA_LIBS="$PHP_READLINE_LIBS"
+# Temporary hack until this gets fixed
+  	PHPDBG_EXTRA_LIBS="-ledit -ltermcap"
   fi
   
   PHP_SUBST(PHP_PHPDBG_CFLAGS)
