Index: php-5.3.29/Zend/Zend.dsp
===================================================================
--- php-5.3.29.orig/Zend/Zend.dsp
+++ php-5.3.29/Zend/Zend.dsp
@@ -255,6 +255,14 @@ SOURCE=.\zend_alloc_canary.c
 # End Source File
 # Begin Source File
 
+SOURCE=.\zend_canary.c
+# End Source File
+# Begin Source File
+
+SOURCE=.\zend_alloc_canary.c
+# End Source File
+# Begin Source File
+
 SOURCE=.\zend_ts_hash.c
 # End Source File
 # Begin Source File
Index: php-5.3.29/Zend/ZendTS.dsp
===================================================================
--- php-5.3.29.orig/Zend/ZendTS.dsp
+++ php-5.3.29/Zend/ZendTS.dsp
@@ -285,6 +285,14 @@ SOURCE=.\zend_alloc_canary.c
 # End Source File
 # Begin Source File
 
+SOURCE=.\zend_canary.c
+# End Source File
+# Begin Source File
+
+SOURCE=.\zend_alloc_canary.c
+# End Source File
+# Begin Source File
+
 SOURCE=.\zend_ts_hash.c
 # End Source File
 # Begin Source File
