diff -urN php5-5.3.10.orig//build/build2.mk php5-5.3.10/build/build2.mk
--- php5-5.3.10.orig//build/build2.mk	2007-01-01 19:32:10.000000000 +0000
+++ php5-5.3.10/build/build2.mk	2012-02-02 22:13:46.000000000 +0000
@@ -52,7 +52,8 @@
 
 aclocal.m4: configure.in acinclude.m4
 	@echo rebuilding $@
-	cat acinclude.m4 ./build/libtool.m4 > $@
+	libtoolize --copy --install --automake --force
+	aclocal
 
 configure: aclocal.m4 configure.in $(config_m4_files)
 	@echo rebuilding $@
