Index: php-5.5.13/ext/standard/info.c
===================================================================
--- php-5.5.13.orig/ext/standard/info.c
+++ php-5.5.13/ext/standard/info.c
@@ -804,6 +804,16 @@ PHPAPI void php_print_info(int flag TSRM
 			zend_html_puts(zend_version, strlen(zend_version) TSRMLS_CC);
 		}
 		php_info_print_box_end();
+
+		/* Dotdeb */
+		php_info_print_box_start(0);
+		if (sapi_module.phpinfo_as_text) {
+			PUTS("This program was built by Dotdeb (http://www.dotdeb.org/).");
+		} else {
+			PUTS("This program was built by <a href=\"http://www.dotdeb.org/\">Dotdeb</a>.");
+		}
+		php_info_print_box_end();
+
 		efree(php_uname);
 	}
 
