diff -urN php5-5.3.10.orig//ext/sqlite/config.m4 php5-5.3.10/ext/sqlite/config.m4
--- php5-5.3.10.orig//ext/sqlite/config.m4	2011-05-15 05:49:34.000000000 +0000
+++ php5-5.3.10/ext/sqlite/config.m4	2012-02-02 22:17:58.000000000 +0000
@@ -84,7 +84,7 @@
     ])
     SQLITE_MODULE_TYPE=external
     PHP_SQLITE_CFLAGS=$pdo_inc_path
-    sqlite_extra_sources="libsqlite/src/encode.c"
+    sqlite_extra_sources=""
   else
     # use bundled library
     PHP_PROG_LEMON
