Description: Adds security notices to php.ini settings
Origin: vendor
Forwarded: not-needed
Last-Update: 2010-01-18

Index: dotdeb-php/php.ini-development
===================================================================
--- dotdeb-php.orig/php.ini-development
+++ dotdeb-php/php.ini-development
@@ -295,6 +295,12 @@ serialize_precision = 17
 ; and below.  This directive makes most sense if used in a per-directory
 ; or per-virtualhost web server configuration file.
 ; http://php.net/open-basedir
+ 
+; NOTE: this is considered a "broken" security measure.
+;       Applications relying on this feature will not receive full
+;       support by the security team.  For more information please
+;       see /usr/share/doc/php5-common/README.Debian.security
+;
 ;open_basedir =
 
 ; This directive allows you to disable certain functions for security reasons.
