Index: php-5.4.27/sapi/fpm/php-fpm.conf.in
===================================================================
--- php-5.4.27.orig/sapi/fpm/php-fpm.conf.in
+++ php-5.4.27/sapi/fpm/php-fpm.conf.in
@@ -160,7 +160,7 @@ group = @php_fpm_group@
 ;                            specific port;
 ;   '/path/to/unix/socket' - to listen on a unix socket.
 ; Note: This value is mandatory.
-listen = 127.0.0.1:9000
+listen = /var/run/php5-fpm.sock
 
 ; Set listen(2) backlog.
 ; Default Value: 128 (-1 on FreeBSD and OpenBSD)
