Index: php-5.3.25.orig/ext/dba/config.m4
===================================================================
--- php-5.3.25.orig.orig/ext/dba/config.m4
+++ php-5.3.25.orig/ext/dba/config.m4
@@ -109,6 +109,10 @@ if test "$PHP_QDBM" != "no"; then
       THIS_PREFIX=$i
       THIS_INCLUDE=$i/include/depot.h
       break
+    elif test -f "$i/include/qdbm/depot.h"; then
+      THIS_PREFIX=$i
+      THIS_INCLUDE=$i/include/qdbm/depot.h
+      break
     fi
   done
 
