Index: dotdeb-php5/ext/pcre/pcrelib/pcre.h
===================================================================
--- dotdeb-php5.orig/ext/pcre/pcrelib/pcre.h
+++ dotdeb-php5/ext/pcre/pcrelib/pcre.h
@@ -42,9 +42,9 @@ POSSIBILITY OF SUCH DAMAGE.
 /* The current PCRE version information. */
 
 #define PCRE_MAJOR          8
-#define PCRE_MINOR          36
+#define PCRE_MINOR          37
 #define PCRE_PRERELEASE     
-#define PCRE_DATE           2014-09-26
+#define PCRE_DATE           2015-04-28
 
 /* When an application links to a PCRE DLL in Windows, the symbols that are
 imported have to be identified as such. When building PCRE, the appropriate
