Index: dotdeb-php5/ext/zlib/zlib.c
===================================================================
--- dotdeb-php5.orig/ext/zlib/zlib.c
+++ dotdeb-php5/ext/zlib/zlib.c
@@ -46,6 +46,18 @@
 #undef gzseek
 #undef gztell
 
+/*
+ * zlib include files can define the following preprocessor defines which rename
+ * the corresponding PHP functions to gzopen64, gzseek64 and gztell64 and thereby
+ * breaking some software, most notably PEAR's Archive_Tar, which halts execution
+ * without error message on gzip compressed archivesa.
+ *
+ * This only seems to happen on 32bit systems with large file support.
+ */
+#undef gzopen
+#undef gzseek
+#undef gztell
+
 ZEND_DECLARE_MODULE_GLOBALS(zlib);
 
 /* {{{ Memory management wrappers */
